src/v2/api/controllers/UserController.ts

```diff
@@ -73,17 +114,46 @@ export class UserController {
     return this.userService.verifyStudent(userId, isCaptain, state);
   }
 
-  @UseGuards(TelegramGuard)
+  @ApiBearerAuth()
+  @ApiOkResponse({
+    type: StudentsResponse,
```

Comment:
This is wrong. Print the another answer
![image](https://github.com/fictadvisor/fictadvisor-api/assets/104054772/90b3d187-190a-4a21-a1c4-e3d0677d1142)